src/App.vue

```diff
@@ -56,10 +56,19 @@ body {
 
 	--light-background: white;
 	--light-secondary-background: whitesmoke;
+	--light-selected-color: rgb(212, 212, 212);
+	--light-hover-color: whitesmoke;
 	--light-font-color: black;
 	--light-border-color: rgb(104, 104, 104);
 
-	--shadow: box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.1);
+	--shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.1);
+}
+
+.light-theme .switch-work-private h4:hover {
+	background-color: var(--light-hover-color);
+}
+.dark-theme .switch-work-private h4:hover {
```

Comment:
this is a terrible way to do white-mode and dark-mode. please do with swapping variables